tests/plugins/test_env_specs.py

```diff
@@ -129,10 +157,14 @@ def test_raises_an_error_if_named_plugin_can_not_be_handled(
     """
     Ensures that an error is raised if the user requests a plugin exists, but can't be handled
     """
-    with pytest.raises(PluginError):
+    with pytest.raises(PluginError) as e:
```

Comment:
Personally prefer using the `match` keyword instead:

```suggestion
    with pytest.raises(
        PluginError,
        match=r"Requested plugin 'rand-spec' is unable to handle environment spec",
    ):
```